config/config.go

```diff
@@ -149,6 +156,23 @@ func FromFlags() (*Config, error) {
 			}
 			cfg.COAKeys[i] = pk
 		}
+	} else if cloudKMSKeys != "" {
+		kmsKeys := strings.Split(cloudKMSKeys, ",")
+		cfg.COACloudKMSKeys = make([]flowGoKMS.Key, len(kmsKeys))
+		for i, key := range kmsKeys {
+			// key has the form "{keyID}@{keyVersion}"
+			keyParts := strings.Split(key, "@")
+			if len(keyParts) != 2 {
+				return nil, fmt.Errorf("wrong format for Cloud KMS key: %s", key)
+			}
+			cfg.COACloudKMSKeys[i] = flowGoKMS.Key{
+				ProjectID:  cloudKMSProjectID,
+				LocationID: cloudKMSLocationID,
+				KeyRingID:  cloudKMSKeyRingID,
+				KeyID:      keyParts[0],
+				KeyVersion: keyParts[1],
+			}
+		}
 	} else {
 		return nil, fmt.Errorf("must either provide coa-key or coa-key-path flag")
```

Comment:
update with kms keys as well

Reply:
Added in https://github.com/onflow/flow-evm-gateway/pull/282/commits/074ba63dc15418e30f97a7fb7b3294d70a20d236